### PR TITLE
HPC new user template email.

### PR DESCRIPTION
**Pull request by redplanks**
gotta inflate those pr counts for hacktoberfest :)

### CODE REVIEW

File: www/templates/hpc-new-user [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Hi {name},
+
+I've added you to the HPC user group, so you can now SSH into hpcctl and start running jobs.
```
Comment (abizer) [inline www/templates/hpc-new-user]: It occurs to me that hpcctl is a really unwieldy name for the single login node, we may want to change the CNAME to hpc.ocf?

Comment (kpengboy) [reply www/templates/hpc-new-user]: At the very least, I would suggest writing an FQDN here.

**Comment by dkess** [comment]
Emails wrap to 72 chars, and that seems to be the convention in this repo.

**Comment by jvperrin** [comment]
Sure, 72 characters then, that works too

**Pull request merged**